src/Theme.php

```diff
@@ -25,7 +25,7 @@
  * <script src="http://example.org/wp-content/themes/my-theme/static/js/all.js"></script>
  * ```
  */
-class Theme extends Core
+class Theme extends Core implements \JsonSerializable
```

Comment:
`use JsonSerializable;`

Since we're making use of `use Something` elsewhere, let's be consistent and always choose imports.

Reply:
👍 Added in [`ec534a1` (#2675)](https://github.com/timber/timber/pull/2675/commits/ec534a1cf262181d3baca22d6c47b393ebdb9452).